src/views/evaluation/Evaluation.js

```diff
@@ -23,35 +23,25 @@ const buttonNav = function(option={}, active="" , specialPlacement = ""){
 
 
 const constructNav = function(option = {}){
-    const currentPage = "Evaluationv2.html", result =[], specialPlacement = "Loginv2.html";
-    for(var i=0;i<option.Link.length;i++){
-        if(option.Link[i] === currentPage){
-            result.push(buttonNav({Link: option.Link[i], Text: option.Text[i]}, "active-page"))
+    const   currentPage = "NewEvaluationv2.html", 
+            specialPlacement = "Loginv2.html",
+            result =[];
+    for(var i=0;i<option.length;i++){
```

Comment:
Let's add a bit of spacing around the (binary) operators. Makes the code more legible and much nicer to look at.